mod-finance/schemas/exchange_rate_source.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Exchange rate source schema",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique identifier of the exchange rate source",
+      "$ref": "../../common/schemas/uuid.json"
+    },
```

Comment:
lets add one more boolean flag for field Specify exchange rate provider.
It will allow to disable this custom functionality without deleting configuration entirely.
![image](https://github.com/user-attachments/assets/2ec1ec0b-3a27-4d80-b90a-0855d69cd2f9)